docs/index.adoc

```diff
@@ -0,0 +1,182 @@
+////
+
+[WARNING]
+====
+When updating the agent token on a previously deployed agent, you will need to remove the `identity.token` file and restart the agent in order to start using the new agent token.
+This file is stored in `~/.local/share/hercules-agent` by default. This will be resolved in https://github.com/hercules-ci/hercules-ci-agent/issues/8[issue #8]
+====
+
+== Deploy with NixOps
+
+1.
+include::autogenerate/Bootstrap.adoc[]
+
+2.
+include::autogenerate/Target.adoc[]
+
+3.
+
+--
+a. Find the account you'd like to deploy the agent in the https://hercules-ci.com[dashboard],
+b. Click the "Agents" button and the button in "Generate token" tab. This produces a private token that should be protected like a password.
+c. Copy the bare token and save it to a plain text file `agent-token.key` in the same directory. Protect the `agent-token.key` file like a password.
+--
+
+4.
+include::autogenerate/Deploy.adoc[]
+
+////
+
+TODO
+
+== Run locally
+
+
+The agent can be run on any machine by installing the agent and invoking the `hercules-agent` command. We recommend running the agent on machines that remain connected to the internet.
+
+This method of running the agent is not officially supported yet: its interface may change in breaking ways.
+
+// TODO is running on your laptop sufficient for some situations?
+
+////
+
+== Troubleshooting
+
+To inspect the agent's local log, run `nixops ssh agent journalctl -u hercules-ci-agent -n 100` to see the last 100 lines.
+
+= FAQ / How To
+
+== My commit status is stuck in Pending - Waiting for agent
+
+Make sure you've deployed an agent. If the agent didn't start, it should leave a message in the system journal. To see the last 100 lines of the journal with NixOps, run `nixops ssh agent journalctl -u hercules-ci-agent -n 100`
+
+Note that an agent can currently only be assigned to a single account. Assigning agents to multiple accounts is on the roadmap, but for now you will have to run multiple agents to build for multiple accounts.
+
+== Can I disable an attribute? I don't want to build that attribute.
+
+You can exclude attributes by putting them inside an attribute set without `recurseIntoAttrs`. This way, you can still use the attribute for other purposes, but it will not show up in CI. For example:
+
+[source,nix]
+----
+# default.nix
+let pkgs = import ./nix {};
+in {
+  inherit (pkgs) myProject;
+  shells = {
+    projectShell = pkgs.mkShell {
+      buildInputs = [ pkgs.nixops ];
+    };
+  };
+}
+----
+
+[source,nix]
+----
+# shell.nix
+(import ./default.nix).shells.projectShell
+----
+
+[source,bash]
+----
+$ nix-build
+/nix/store/...-hello-2.10
+$ nix-shell --run 'nixops --version'
+NixOps 1.6.1
+----
+
+== My attribute doesn't show up in the evaluation
+
+Is it in a nested attribute set? Make sure you call `pkgs.recurseIntoAttrs` on all levels of nested attrsets.
+
+Does `nix-build` build the attribute? If it doesn't, Hercules CI shouldn't build it either. Otherwise, please contact us.
```

Comment:
```suggestion
Does `nix-build` build the attribute with specifying `-A`? If it doesn't, Hercules CI shouldn't build it either. Otherwise, please contact us.
```